scripts/documentation/docs-html.js

```diff
@@ -9,7 +9,9 @@ documentation
   .build(['types/*.js', 'services/**/*/lib/transform.js'], {})
   .then(comments =>
     documentation.formats.html(comments, {
-      'project-name': 'EUBFR API docs',
+      'project-name': 'EUBFR ETL documentation',
+      'project-description':
+        'Information about transform functions used throughout the project.',
```

Comment:
Could be nice to add `project-url` and to use this parameter as the href of the logo (currently pointing to `/example`)